src/client/hud.h

```diff
@@ -125,6 +128,8 @@ class Hud
 
 	scene::SMeshBuffer m_rotation_mesh_buffer;
 
+	u8 m_block_bounds_mode = 0;
```

Comment:
Please change this to an enum value for clarity.
```C++
enum BlockBoundMode {
	BBM_??,
	BBM_??,
	BBM_??
};
BlockBoundMode m_block_bounds_mode = ???;
```

Reply:
As long as it's okay to do `m_block_bounds_mode = static_cast<BlockBoundsMode>(m_block_bounds_mode + 1)` as a way to cycle through them...